themes/doom-kanagawa-theme.el

```diff
@@ -0,0 +1,178 @@
+;;; doom-kanagawa-theme.el --- inspired by rebelot/kanagawa.nvim and others -*- lexical-binding: t; no-byte-compile: t; -*-
+;;
+;; Added: June 4 2023
+;; Author: Anskrevy <https://github.com/Anskrevy
+;; Maintainer:
+;; Source: ????
```

Comment:
```suggestion
;; Source: https://github.com/rebelot/kanagawa.nvim
```

Reply:
What about multiple sources? Rebelot's theme seems to be the most well known, but this theme does not a directly port it. I picked the bits I liked between that theme, [jasonm23/emacs-theme-kanagawa](https://github.com/jasonm23/emacs-theme-kanagawa), and [NvChad kanagawa](https://github.com/NvChad/base46/blob/v2.0/lua/base46/themes/kanagawa.lua).

Reply:
Those are all derived works from rebelot, so there should at least be this one. Feel free to add extra source fields if you feel like attribution is necessary though

Reply:
Good point, didn't think of it that way. Updated to reflect the source.